test/upgrade-v2/onchain_upgrade_v2_spredsheet_app_script.js

```diff
@@ -0,0 +1,327 @@
+// This script is AppScript for spreadsheets
+// for spreadsheets.
+// It is used in the following spreadsheets.
+// https://cauchye.slack.com/files/U8W7Y7VHT/F04DM12FPKP/token_distribution_on_upgrade_in_the_middle_of_december
+const ERROR_CODE = {
+    none: 0,
+    no_address: 1, 
+    address_mismatched_discord_usernames: 2,
+    amount_0: 3
+  }
+  const ERROR_MSG = [
+    "no error",
+    "target no ununifi address.",
+    "Same address but mismatched Discord usernames.",
+    "target amount 0."
+  ]
+  // input data index
+  const WALLET_ADDRESS = 0;
+  const DISCORD_ID = 1;
+  const VESTING_AMOUNT_UGUU = 2;
+  const VALIDATOR_AMOUNT_UGUU = 3;
+  const VESTING_STARTS = 5;
+  const VESTING_ENDS = 6;
+  
+  function main() {
+    const SpredSheet = SpreadsheetApp.getActiveSpreadsheet();
```

Comment:
"spredsheet" is supposed to be "spreadsheet", innit?

Reply:
sorry. 
fix typo "SpredSheet" -> "spreadsheet"